### PR TITLE
cmake: small fixes to install

### PR DESCRIPTION
**Pull request by cmcfarlen**

### CODE REVIEW

File: src/api/CMakeLists.txt [hunk 1]
```diff
@@ -51,7 +51,7 @@ target_link_libraries(
 )
 
 install(
-  TARGETS tsapi tsapibackend
```
Comment (cmcfarlen) [inline src/api/CMakeLists.txt:54]: I should have removed this in the prior PR

File: cmake/post_install.cmake [hunk 2]
```diff
@@ -17,8 +17,7 @@
 
 if("$ENV{USER}" STREQUAL root)
   foreach(DIR ${CHOWN_DIRS})
-    message(STATUS "Changing ${DIR} ownership to ${OWNER_USER}:${OWNER_GROUP}")
+    message(STATUS "Changing $ENV{DESTDIR}${DIR} ownership to ${OWNER_USER}:${OWNER_GROUP}")
```
Comment (JosiahWI) [inline cmake/post_install.cmake:20]: It shouldn't have `${CMAKE_INSTALL_PREFIX}` as well like the line below it?

Comment (cmcfarlen) [reply cmake/post_install.cmake:20]: Actually, the other should look like this one! I missed this picking from internal code.  Thanks!

**Review by JosiahWI — APPROVED** [review]
This also makes the CMakeUserPresets in .gitignore relative to the .gitignore location.

Looks good.

**Review by SolidWallOfCode — APPROVED** [review]

**Pull request merged**